src/test/java/redis/clients/jedis/util/JedisURIHelperTest.java

```diff
@@ -1,15 +1,14 @@
 package redis.clients.jedis.util;
 
+import static org.junit.Assert.*;
 import static redis.clients.jedis.util.JedisURIHelper.*;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
-
 import java.net.URI;
 import java.net.URISyntaxException;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 import redis.clients.jedis.RedisProtocol;
+import redis.clients.jedis.exceptions.JedisException;
```

Comment:
```suggestion
import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.emptyString;
```
Because of IllegalArgumentException + new test for emptyPassword